### ISSUE TITLE
Example project does not run.

### ISSUE THREAD

**Issue by UmairSaqibBhutta** [MOUKZ/chat_package#1]
Warning: Mapping new ns http://schemas.android.com/repository/android/common/02 to old ns http://schemas.android.com/repository/android/common/01
Warning: Mapping new ns http://schemas.android.com/repository/android/generic/02 to old ns http://schemas.android.com/repository/android/generic/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/addon2/02 to old ns http://schemas.android.com/sdk/android/repo/addon2/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/repository2/02 to old ns http://schemas.android.com/sdk/android/repo/repository2/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/sys-img2/02 to old ns http://schemas.android.com/sdk/android/repo/sys-img2/01
/E:/flutter_windows_3.0.2-stable/flutter/.pub-cache/hosted/pub.dartlang.org/platform-3.0.2/lib/src/interface/local_platform.dart:46:19: Error: Member not found: 'packageRoot'.
      io.Platform.packageRoot; // ignore: deprecated_member_use
                  ^^^^^^^^^^^
2

FAILURE: Build failed with an exception.

* Where:
Script 'E:\flutter_windows_3.0.2-stable\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 1156

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command 'E:\flutter_windows_3.0.2-stable\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 30s
Exception: Gradle task assembleDebug failed with exit code 1

**Comment by MOUKZ** [comment]
sorry for the late reply 
you can check the beta version now
and I'm still trying to add new features to it

**Comment by UmairSaqibBhutta** [comment]
I think only images can be shared. Please try to add other things like documents or videos etc.

**Comment by MOUKZ** [comment]
you are correct, I will add video support and add better documentation to it as soon as possible
thank you so much for the feedback <3
